eventmesh-runtime/build.gradle

```diff
@@ -24,14 +24,14 @@ dependencies {
     api 'io.prometheus:simpleclient_httpserver'
 
     implementation project(":eventmesh-connector-plugin:eventmesh-connector-api")
-    implementation project(":eventmesh-connector-plugin:eventmesh-connector-standalone")
+    implementation project(":eventmesh-connector-plugin:eventmesh-connector-rocketmq")
```

Comment:
Set default connector as standalone is ok in my opinion.

Reply:
Agree with you, done.